### PR TITLE
Eddoliveira patch escape field name

### PR DESCRIPTION
**Pull request by eddoliveira**

### CODE REVIEW

**Review by shaffooo — CHANGES_REQUESTED** [review]
The `re.escape` change looks good but we don't want to apply newlines to the log. People will need to enable multiline log parser settings to consume the logs which is not desirable.

File: nplusone/base.py [hunk 1]
```diff
@@ -23,8 +23,8 @@ class NPlusOne:
     will detect that and print a debug message to alert developer.
     """
 
-    WARNING_MSG_FORMAT = 'Possible N+1 for model: {model}, field: {field}, relationship: {relationship},'\
-        'file: {file}, function: {function}, line: {line}, statement: {statement}'
+    WARNING_MSG_FORMAT = '\n*** Possible N+1 for model: {model}, field: {field},\nrelationship: {relationship},'\
+        '\nfile: {file}, \nfunction: {function}, line: {line}, statement: {statement}\n'
```
Comment (shaffooo) [inline nplusone/base.py]: Can you please revert this change?

Comment (eddoliveira) [reply nplusone/base.py]: Sorry, that was just for me to use locally, I forgot to remove before comitting.
I've updated the branch.

Thanks for answering!

**Review by shaffooo — APPROVED** [review]
Looks good
